FreeRADIUS: add doc (#853)
SEKOIA-IO/documentation: docs/xdr/features/collect/integrations/application/freeradius.md

uuid: aeb7d407-db57-44b2-90b6-7df6738d5d7f
name: FreeRADIUS
type: intake

## Overview

FreeRADIUS is an implementation of RADIUS, a protocol for remote user Authentication, Authorization and Accounting.

> In this documentation we will explain how to collect and send FreeRADIUS logs to SEKOIA.IO.

Reviewer: ```suggestion
In this documentation, you will learn how to collect and send FreeRADIUS logs to SEKOIA.IO.
```

{!_shared_content/operations_center/detection/generated/suggested_rules_aeb7d407-db57-44b2-90b6-7df6738d5d7f_do_not_edit_manually.md!}

{!_shared_content/operations_center/integrations/generated/freeradius_do_not_edit_manually.md!}

## Configure

This setup guide will lead you into forwarding FreeRADIUS's logs to SEKOIA.IO

### Prerequisites

An internal log concentrator (Rsyslog) is required to collect and forward events to SEKOIA.IO.

### Create the intake in SEKOIA.IO

Go to the [intake page](https://app.sekoia.io/operations/intakes) and create a new intake from the format `FreeRADIUS`. Copy the intake key.

### Transport to SEKOIA.IO

Please consult the [Rsyslog Transport](../../../ingestion_methods/rsyslog/) documentation to forward these logs to SEKOIA.IO

### Enable Syslog forwarding for FreeRADIUS

On the system hosting the FreeRADIUS:

1. Edit the file `/etc/freeradius/radiusd.conf` and update the log section:

	```bash
	$ vi /etc/freeradius/radiusd.conf
	log {
	  destination = syslog
	  syslog_facility = local1
	  stripped_names = no
	  auth = yes

	  # don't logs password
	  auth_badpass = no 
	  auth_goodpass = no
	}
	```

> 2. Foward logs to the concentrator:

Reviewer: ```suggestion
2. Forward logs to the concentrator:
```
